BugsnagBundle.php

```diff
@@ -12,4 +14,16 @@ class BugsnagBundle extends Bundle
      * @return string
      */
     const VERSION = '1.0.0';
+
+    /**
+     * {@inheritdoc}
```

Comment:
A better description would be "Setup the callback registering pass." or something.